lib/algolia_index.rb

```diff
@@ -0,0 +1,38 @@
+require_relative 'algolia_index/models/base'
+require_relative 'algolia_index/models/firm'
+require_relative 'algolia_index/models/adviser'
+require_relative 'algolia_index/models/office'
+
+require_relative 'algolia_index/serializers/firm_serializer'
+require_relative 'algolia_index/serializers/adviser_serializer'
+require_relative 'algolia_index/serializers/office_serializer'
+
+module AlgoliaIndex
+  INDICES = {
+    advisers: "firm-advisers#{'-test' if Rails.env.test?}",
+    offices: "firm-offices#{'-test' if Rails.env.test?}"
+  }.freeze
+
+  class << self
+    def handle_update(klass:, id:, firm_id:)
+      index = "AlgoliaIndex::#{klass}".constantize.new(
```

Comment:
Still think this is something more like 'indexed_record' than 'index'. WDYT?

Reply:
PR's looking good btw, only have this question left.

Reply:
Looks good to me 👍

Reply:
♻️ done! :)